rubric_visualization/settings/base.py

```diff
@@ -226,7 +224,8 @@
 ]
 
 SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
-SESSION_COOKIE_SAMESITE = None
+SESSION_COOKIE_SAMESITE = 'None'
```

Comment:
Nice to see this is now supported in Django 3.1 without requiring any work-aroudns, although I'm a bit surprised that it seems the django team is not going to backport it to 2.2 LTS , at least according to [this ticket](https://code.djangoproject.com/ticket/31933).

Reply:
Ah, what a bummer. At this point I don't understand how our older django apps are still functioning iframed in canvas. Is there some workaround already in place on the `atg-(dev/prod)-djangoapp-appserver`s?

Reply:
I believe since most of our older django apps share the `harvard.edu` domain with canvas, they are considered the same site [eTLD + 1](https://web.dev/same-site-same-origin/#site), so the cookies work as expected when embedded in an iframe. For local development on these older apps, I've started using local.tlt.harvard.edu instead of localhost for that reason.

Reply:
Awesome, thanks